test/run_e2e_test.py

```diff
@@ -193,28 +226,32 @@ def check_for_expected_topic(expected_topic: str) -> None:
         raise SystemMetricsEnd2EndTestException('Unable to find expected topic: ' + str(output))
 
 
-@retry(stop_max_attempt_number=DEFAULT_MAX_ATTEMPTS,
-       wait_exponential_multiplier=DEFAULT_WAIT_EXPONENTIAL_MULTIPLIER,
-       wait_exponential_max=DEFAULT_MAX_EXPONENTIAL_WAIT_MILLISECONDS)
+@retry(stop_max_attempt_number=DEFAULT_MAX_ATTEMPTS, wait_fixed=DEFAULT_FIXED_WAIT_MILLISECONDS)
```

Comment:
Is the wait no longer needed b/c you're already checking the expected number of msgs to receive?

Reply:
Right, this wait is longer than ```TIMEOUT_SECONDS``` on line 210 in order to let spin complete. I'll comment so that's clear.